### PR TITLE
add task solution

### PR DESCRIPTION
**Pull request by ArsenSkandalenko**
https://arsenskandalenko.github.io/layout_product-cards/

### CODE REVIEW

File: src/index.html [hunk 1]
```diff
@@ -6,8 +6,47 @@
...
+      <img
+        class="card__image"
+        src="./images/imac.jpeg"
+        alt="APPLE A1419 iMac 27 inches"
+      >
+
+      <h3 class="card__product-name">
+          APPLE A1419 iMac 27" Retina
+          5K Monoblock (MNED2UA/A)
+      </h3>
```
Comment (witflash) [inline src/index.html:25]:
```suggestion
      <h3 class="card__product-name">
        APPLE A1419 iMac 27" Retina
        5K Monoblock (MNED2UA/A)
      </h3>
```
You have too many spaces. Check it, please

File: src/styles/index.scss [hunk 2]
```diff
@@ -1,3 +1,125 @@
+
```
Comment (witflash) [inline src/styles/index.scss:1]: Redundant empty line. Just remove it at all

File: src/styles/index.scss [hunk 3]
```diff
@@ -1,3 +1,125 @@
...
   margin: 0;
 }
+
+.card {
+width: 198px;
+height: 406px;
+background-color: $color-white;
+border: 1px solid #f3f3f3;
+border-radius: 5px;
+margin: 20px 48px;
```
Comment (witflash) [inline src/styles/index.scss:18]:
```suggestion
.card {
  width: 198px;
  height: 406px;
  background-color: $color-white;
  border: 1px solid #f3f3f3;
  border-radius: 5px;
  margin: 20px 48px;
```

All properties inside the rule block should have an indentation.

File: src/styles/index.scss [hunk 4]
```diff
@@ -1,3 +1,125 @@
...
+
+  &:hover {
+    color: $color-blue;
+    border: 1px solid $color-blue;
+    background-color: $color-white;
+  }
+}
+
+
+
```
Comment (witflash) [inline src/styles/index.scss:124]: Too many empty lines at the end of the file. You have to leave just one empty line at the end of every file.

**Review by witflash — CHANGES_REQUESTED** [review]
Check my comments, please.

And check if your deployment was successful. 
For now, your DEMO and TEST links don't work :( 
Maybe you forgot to deploy your project. Or you had some errors during deployment. Check it, please.

**Review by maxim2310 — CHANGES_REQUESTED** [review]
Your demo link still not working, fix it plz and add demo and test link in comment to Pull request

**Review by maxim2310 — CHANGES_REQUESTED** [review]
Please add your demo link to PR!!!

File: src/styles/index.scss [hunk 5]
```diff
@@ -1,3 +1,120 @@
...
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+.card {
+  width: 198px;
+  height: 406px;
```
Comment (anastasiiavorobiova) [inline src/styles/index.scss:13]:
```suggestion

```

File: src/styles/index.scss [hunk 6]
```diff
@@ -1,3 +1,120 @@
...
+  background-color: $color-white;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  margin: 20px 48px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    padding: 32px 19px;
+
```
Comment (anastasiiavorobiova) [inline src/styles/index.scss:23]:
```suggestion
```

File: src/styles/index.scss [hunk 7]
```diff
@@ -1,3 +1,120 @@
...
+    height: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &__star:not(:last-of-type) {
+    margin-right: 4px;
+  }
+
+  &__star:nth-child(-n + 4) {
```
Comment (anastasiiavorobiova) [inline src/styles/index.scss:65]: The stars quantity should depend on the modifier

File: src/styles/index.scss [hunk 8]
```diff
@@ -1,3 +1,120 @@
...
+    font-size: 16px;
+    line-height: 18px;
+    font-weight: 700;
+    color: $color-dark;
+  }
+}
+
+.button {
+  border: 0;
+  width: 166px;
```
Comment (anastasiiavorobiova) [inline src/styles/index.scss:102]:
```suggestion
  width: 100%;
```

**Review by anastasiiavorobiova — CHANGES_REQUESTED** [review]
Great work! Almost done!

File: readme.md [hunk 9]
```diff
@@ -20,8 +20,8 @@ Create a pages with product card using `flexbox`, `BEM` and `SCSS` based on [thi
 
 ❗️ Replace `<your_account>` with your Github username and copy the links to `Pull Request` description:
 
-- [DEMO LINK](https://<your_account>.github.io/layout_product-cards/)
-- [TEST REPORT LINK](https://<your_account>.github.io/layout_product-cards/report/html_report/)
+- [DEMO LINK](https://ArsenSkandalenko.github.io/layout_product-cards/)
+- [TEST REPORT LINK](https://ArsenSkandalenko>.github.io/layout_product-cards/report/html_report/)
```
Comment (etojeDenys) [inline readme.md:24]:
```suggestion
- [TEST REPORT LINK](https://ArsenSkandalenko.github.io/layout_product-cards/report/html_report/)
```

File: src/styles/index.scss (hunk 7 above)
Comment (etojeDenys) [inline src/styles/index.scss:65]: move the stars to another file as separate BEM block

File: src/styles/index.scss (hunk 5 above)
Comment (etojeDenys) [reply src/styles/index.scss:13]: image

File: src/styles/index.scss (hunk 7 above)
Comment (etojeDenys) [reply src/styles/index.scss:65]: image

File: src/styles/index.scss (hunk 8 above)
Comment (etojeDenys) [reply src/styles/index.scss:102]: image

**Review by etojeDenys — CHANGES_REQUESTED** [review]
fix all prev comments, and try to fix your tests report link

feel free to ask for some help with that in the chat

**Review by vadiimvooo — CHANGES_REQUESTED** [review]
Seems like you didn't fix previous comments. Feel free to ask for help in fe_chat,

**Pull request closed without merging**